### PR TITLE
Fixed [Issue 867] Simplified code in setup.py

### LINKED ISSUE #867: Simplify code in setup.py
Issues found:
* **get_version**: `RuntimeError` is unreachable
* **get_dependencies**: If statements can be simplified

### PR DESCRIPTION
**Pull request by theodesp**
Fixes #867

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,20 +9,22 @@
...
-        exec(f.read(), locals)
-        return locals['VERSION']
-    raise RuntimeError('No version info found.')
+    try:
+        with open(os.path.join(basedir, 'rq/version.py')) as f:
+            locals = {}
+            exec(f.read(), locals)
+            return locals['VERSION']
+    except FileNotFoundError:
+        raise RuntimeError('No version info found.')
```
Comment (theodesp) [inline setup.py]: Now it will properly throw `RuntimeError`

File: setup.py [hunk 2]
```diff
@@ -56,13 +58,13 @@ def get_dependencies():
     },
     classifiers=[
         # As from http://pypi.python.org/pypi?%3Aaction=list_classifiers
-        #'Development Status :: 1 - Planning',
-        #'Development Status :: 2 - Pre-Alpha',
-        #'Development Status :: 3 - Alpha',
-        #'Development Status :: 4 - Beta',
+        # 'Development Status :: 1 - Planning',
```
Comment (theodesp) [inline setup.py]: Pep8

File: setup.py (hunk 1 above)
Comment (rpkilby) [reply setup.py]: Why is it necessary to throw a `RuntimeError` instead of a `FileNotFoundError`?

File: setup.py [hunk 3]
```diff
@@ -9,20 +9,22 @@
...
+            locals = {}
+            exec(f.read(), locals)
+            return locals['VERSION']
+    except FileNotFoundError:
+        raise RuntimeError('No version info found.')
 
 
 def get_dependencies():
     deps = ['redis >= 2.7.0', 'click >= 5.0']
     if sys.version_info < (2, 7) or \
```
Comment (selwin) [inline setup.py]: We no longer support Python 2.6 and below. Could you please change this part?

**Comment by theodesp** [comment]
Fixed and ready to merge

**Comment by selwin** [comment]
Thanks!

**Pull request merged**